src/Query/Builder.php

```diff
@@ -614,11 +614,7 @@ public function orderBy($column, $direction = 'asc')
         return $this;
     }
 
-    /**
-     * @param list{mixed, mixed}|CarbonPeriod $values
```

Comment:
This signature was causing psalm error. Param type incompatible with the parent method.